packages/amplify-nodejs-runtime-provider/src/index.ts

```diff
@@ -20,8 +23,13 @@ export const functionRuntimeContributorFactory: FunctionRuntimeContributorFactor
     checkDependencies: () => Promise.resolve({ hasRequiredDependencies: true }),
     package: params => packageResource(params, context),
     build: params => buildResource(params),
-    invoke: params => {
-      throw new Error('not yet implemented');
-    },
+    invoke: params =>
+      invoke({
+        packageFolder: path.join(params.srcRoot, 'src'),
```

Comment:
This is still wrong. 'src' is only root for legacy packages, for new node functions we decided to have the package at the root level, runtime is not enforcing it. This is to get rid of src/src in case a nodejs lambda is TypeScript which naturally has an src folder.

Reply:
I know we want to remove it, but for now I'm leaving it as-is to ensure backwards compatibility. Supporting the new directory structure can come later